documentation/src/main/asciidoc/public/reference/_indexing-explicit.adoc

```diff
@@ -0,0 +1,281 @@
+[[indexing-explicit]]
+= [[mapper-orm-indexing-manual]] [[manual-index-changes]] Explicit indexing
+
+[[indexing-explicit-basics]]
+== [[mapper-orm-indexing-manual-basics]] [[search-batchindex]] Basics
+
+While <<listener-triggered-indexing,listener-triggered indexing>> and
+the <<indexing-massindexer,`MassIndexer`>>
+or <<mapper-orm-indexing-jakarta-batch,the mass indexing job>>
+should take care of most needs,
+it is sometimes necessary to control indexing manually.
+
+The need arises in particular when <<listener-triggered-indexing,listener-triggered indexing>> is <<mapping-annotations,disabled>>
+or simply not supported (e.g. <<mapper-pojo-standalone-indexing-listener-triggered,with the Standalone POJO Mapper>>),
+or when entity changes happen listener-triggered indexing did not or cannot detect --
+<<limitations-changes-in-session,such as JPQL/SQL `insert`, `update` or `delete` queries>>.
```

Comment:
```suggestion
or when listener-triggered indexing cannot detect entity changes happening --
<<limitations-changes-in-session,e.g. JPQL/SQL `insert`, `update` or `delete` queries>>.
```
Maybe something like that? saying `listener-triggered indexing did not detect` feels like it should've but it didn't and  that feels more like  a 🪳  😃